### PR TITLE
Azure Stack: add authorization RP

### PR DESCRIPTION
**Pull request by yugangw-msft**
For review. PR will become merge-able once the multi-api SDK gets finalized

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py [hunk 1]
```diff
@@ -102,6 +104,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_POLICY: '2015-10-01-preview',
         ResourceType.MGMT_RESOURCE_RESOURCES: '2016-02-01',
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
+        ResourceType.MGMT_AUTHORIZATION: '2015-07-01',
```
Comment (bganapa) [inline src/azure-cli-core/azure/cli/core/profiles/_shared.py]: This APi version is correct, just confirming

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py [hunk 2]
```diff
@@ -89,6 +90,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_POLICY: '2017-06-01-preview',
         ResourceType.MGMT_RESOURCE_RESOURCES: '2017-05-10',
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
+        ResourceType.MGMT_AUTHORIZATION: '2018-01-01-preview',
```
Comment (lmazuel) [inline src/azure-cli-core/azure/cli/core/profiles/_shared.py]: Replace by:
```python
ResourceType.MGMT_AUTHORIZATION: SDKProfile('2018-01-01-preview',
    'classic_administrators': '2015-06-01'
)
```
This says "use 2018-01-01-preview for everything, except classic_administrators use 2015-06-01"

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py (hunk 1 above)
Comment (lmazuel) [inline src/azure-cli-core/azure/cli/core/profiles/_shared.py]: Replace by:
```python
ResourceType.MGMT_AUTHORIZATION: SDKProfile('2015-07-01')
```
This implies not classic_administrators support in Stack. If it's not true, add the classic_admin admin key like I did above

**Comment by yugangw-msft** [comment]
#6781 would cover it

**Pull request closed without merging**